### PR TITLE
modified cleanup for the OVS for 3.2.1 and 4.0

### PR DESCRIPTION
**Pull request by xdel**
Accept with https://mirantis.jira.com/browse/OSCI-779 completion. Please also mind of backporting upstart diff to the 3.2.1 and to the Grizzly.

### CODE REVIEW

**Comment by xenolog** [comment]
-1

File: deployment/puppet/openstack/files/quantum-agent-ovs [hunk 1]
```diff
@@ -232,30 +232,17 @@ quantum_ovs_agent_monitor() {
...
 clean_up() {
-	
-	ocf_log info "cleaning up integration bridge"
-	ovs-vsctl -- --if-exists del-br ${OCF_RESKEY_segmentation_bridge}
+	ocf_log info "Cleaning OVS topology..."
+	mkdir -p /var/run/neutron-ovs-cleanup
+	chown neutron:neutron /var/run/neutron-ovs-cleanup
+	mkdir -p /var/lock/neutron-ovs-cleanup
+	chown neutron:root /var/lock/neutron-ovs-cleanup
+	/usr/bin/neutron-ovs-cleanup --config-file=/etc/neutron/plugins/openvswitch/ovs_neutron_plugin.ini --log-file=/var/log/neutron/neutron-ovs-cleanup.log
```
Comment (aglarendil) [inline deployment/puppet/openstack/files/quantum-agent-ovs:240]: please do this using sudo

Comment (xdel) [inline deployment/puppet/openstack/files/quantum-agent-ovs:240]: https://github.com/redhat-openstack/openstack-neutron/blob/master/neutron-ovs-cleanup.service

**Comment by xenolog** [comment]
Andrey, we  need only make RC-script for clean ovs-bridges. This script must be executed on boot hardware. Only on boot, not on restart pvs-agent.

**Comment by xdel** [comment]
As I remember cleanup should be issued before every restart too. Will verify soon on 3.2 release.

**Comment by xenolog** [comment]
no. only on boot.

**Pull request closed without merging**